.github/workflows/acceptance_test.yml

```
@@ -33,7 +33,7 @@ jobs:
       - name: "[Pull Request] Get commit message"
         if: github.event_name == 'pull_request'
         id: pr_get_commit_message
-        run: echo ::set-output name=pr_commit_message::$(git log --format=%B -n 1 HEAD^2)
+        run: echo "pr_commit_message=$(git log --format=%B -n 1 HEAD^2)" >> $GITHUB_OUTPUT
```

The Github action is failing. Try the line below to add quotes to the value in case the commit message contains multiple words: 
```suggestion
        run: echo "pr_commit_message=\"$(git log --format=%B -n 1 HEAD^2)\"" >> $GITHUB_OUTPUT
```

---

Ah! Shoot you're right, didn't see the ".

---

Looks ok now: https://github.com/MobilityData/gtfs-validator/actions/runs/6177475232